lib/auth/methods.go

```diff
@@ -463,6 +463,14 @@ func (s *Server) AuthenticateSSHUser(req AuthenticateSSHRequest) (*SSHLoginRespo
 		authority,
 	}
 
+	sourceIP := ""
+	if checker.PinSourceIP() {
+		host, err := utils.Host(req.ClientMetadata.RemoteAddr)
```

Comment:
If the versions of the Proxy and Auth server differ, there's a chance that `req.ClientMetadata` will be nil, causing a panic.

Reply:
good point, fixed